setup-services.sh

```diff
@@ -32,11 +32,12 @@ sudosed() {
 
 # if agama is already running -> stop it
 $SUDO systemctl list-unit-files agama.service &>/dev/null && $SUDO systemctl stop agama.service
+$SUDO systemctl list-unit-files agama-web-server.service &>/dev/null && $SUDO systemctl stop agama-web-server.service
 
 # Ruby services
 
 # Packages required for Ruby development (i.e., bundle install).
-$SUDO zypper --non-interactive --gpg-auto-import-keys install \
```

Comment:
The `--gpg-auto-import-keys` option is only needed when installing packages from unknown 3rd party repositories. It is not needed when using the official openSUSE repositories. (It could actually accidentally accept some forged GPG key...)